sections/Discussion.tex

```diff
@@ -1 +1,24 @@
 \section{Discussion}\label{sec:discussion}
+In this section, we analyze and describe the results, we get from running our experiment.
+
+\gls{pr} using our adjacency-matrix successfully improved upon the results of other models in many cases.
+This is worth noting as our adjacency matrix was constructed based on similarity of the documents' topic-distributions, rather than any relation embedded in the documents themselves, such as articles referencing each other as with \cite{yang2009topic}.
```

Comment:
```suggestion
This is worth noting as our adjacency matrix was constructed based on similarity of the documents' topic-distributions, rather than any relation embedded in the documents themselves, such as scientific articles referencing each other as with \cite{yang2009topic}.
```